mlflow/models/evaluation/base.py

```diff
@@ -74,8 +74,8 @@ class EvaluationMetric:
 
             def eval_fn(
                 eval_df: Union[pandas.Dataframe, pyspark.sql.DataFrame],
-                builtin_metrics: Dict[str, float],
-            ) -> float:
```

Comment:
I think this should be `Union[float, MetricValue]`. Users shouldn't have to construct `MetricValue` if they just want to return a float (we should construct `MetricValue` automatically for them)